lua/nordic/groups/integrations.lua

```diff
@@ -97,39 +97,83 @@ function M.get_groups()
     G.MiniStatuslineFileinfo = { bg = LC.normal.b.bg, fg = LC.normal.b.fg }
     G.MiniStatuslineInactive = { bg = LC.inactive.a.bg, fg = LC.inactive.a.fg, bold = LC.inactive.a.gui == 'bold' }
 
+    -- Tree Groups
+    G.TreeNormal = { fg = C.fg, bg = C.bg }
+    G.TreeNormalNC = { fg = C.fg, bg = C.bg }
+    G.TreeRootFolder = { fg = C.gray4 }
+    G.TreeRootName = { fg = C.fg, bold = true }
+    G.TreeFileIcon = { fg = C.blue2 }
+    G.TreeFileNameOpened = { fg = C.fg }
+    G.TreeSpecialFile = { fg = C.magenta.bright }
+    G.TreeGitConflict = { fg = C.magenta.bright }
+    G.TreeGitModified = { fg = C.git.change }
+    G.TreeGitDirty = { fg = C.gray4 }
+    G.TreeGitAdded = { fg = C.git.add }
+    G.TreeGitNew = { fg = C.gray4 }
+    G.TreeGitDeleted = { fg = C.gray4 }
+    G.TreeGitStaged = { fg = C.gray4 }
+    G.TreeGitUntracked = { fg = C.orange.base }
+    G.TreeTitleBar = { fg = C.black0, bg = C.blue2, bold = true }
+    G.TreeFloatBorder = { fg = C.blue2, bg = C.gray0}
+    G.TreeCursorLine = { bg = C.bg_selected }
+    G.TreeCursor = { bg = C.none, fg = C.none }
+    G.TreeFolderIcon = { fg = C.yellow.dim }
+    G.TreeIndentMarker = { fg = C.gray4 }
+    G.TreeSymlink = { fg = C.blue2 }
+    G.TreeFolderName = { fg = C.blue1 }
+    G.TreeWinSeparator = { fg = C.bg_dark, bg = C.bg }
```

Comment:
Thanks for your reply, Alex!
Couldn't reply earlier because I had some other things to do.
I'll see what I can do and I'll try to finish it as soon as I can.

Reply:
i can't find the existing window separator

Reply:
just use: `link = "WinSeparator"`
see definition: https://github.com/AlexvZyl/nordic.nvim/blob/1ee4044077059b34eacd18d93613495a602c131f/lua/nordic/groups/native.lua#L135

Reply:
Sorry, should just have told you what the hl is :joy:

Reply:
Np, I'm thinking now how could I not find the window separator, it was there the whole time...😅